crates/test-utils/src/lib.rs

```diff
@@ -728,19 +729,17 @@ fn execute_runtime_functions(executor: &mut Executor, runtime: &Runtime) -> (Exi
         .expect("failed to compile Yul");
     let bytecode = hex::decode(contract_bytecode.bytecode).expect("failed to decode bytecode");
 
-    if let evm::Capture::Exit((reason, _, output)) = executor.create(
+    let evm::Capture::Exit((reason, _, output)) = executor.create(
         address(DEFAULT_CALLER),
         evm_runtime::CreateScheme::Legacy {
             caller: address(DEFAULT_CALLER),
         },
         U256::zero(),
         bytecode,
         None,
-    ) {
-        (reason, output)
-    } else {
-        panic!("EVM trap during test")
```

Comment:
Clippy complained that this `else` block never gets reached and that the surrounding `if let` was unnecessary